apps/galaxy/src/modules/auth/SignUpForm/SignUpFrom.schema.tsx

```diff
@@ -50,10 +50,7 @@ export const CompanyInfoFormSchema = yup
       .string()
       .required("The contact number is required")
       .phone("AU", "Please enter a valid AU phone number"),
-    businessAddress: yup
-      .string()
-      .required("The business address is required")
-      .min(10, "You need to provide a valid business address"),
+    businessAddress: yup.string(),
```

Comment:
It does not work if you still keep yup.string()

Reply:
I tried several times it worked somehow